app/renderer/components/preferences/payment/ledgerBackup.js

```diff
@@ -98,23 +96,17 @@ const styles = StyleSheet.create({
     margin: `${globalStyles.spacing.dialogInsideMargin} auto`
   },
 
-  ledgerBackupContent__copyKey_second: {
-    marginBottom: 0
-  },
-
   ledgerBackupContent__copyKey__key: {
-    marginLeft: `calc(${globalStyles.spacing.dialogInsideMargin} * 2)`
-  },
-
-  ledgerBackupContent__copyKey__key__header: {
-    marginBottom: globalStyles.spacing.modalPanelHeaderMarginBottom
-  },
+    marginLeft: `calc(${globalStyles.spacing.dialogInsideMargin} * 2)`,
 
-  ledgerBackupContent__copyKey__key__phrase: {
+    // See syncTab.js
```

Comment:
We could create a common component to avoid style inconsistency, like `PassPhraseContainer`.